superchain/params.go

```diff
@@ -0,0 +1,56 @@
+package superchain
+
+import "math/big"
+
+type L2OOParams struct {
+	StartingBlockNumber       *big.Int // The number of the first L2 block.
+	StartingTimestamp         *big.Int // The timestamp of the first L2 block.
+	SubmissionInterval        *big.Int // Interval in blocks at which checkpoints must be submitted.
+	L2BlockTime               *big.Int // The time per L2 block, in seconds.
+	FinalizationPeriodSeconds *big.Int // The minimum time (in seconds) that must elapse before a withdrawal can be finalized.
+}
```

Comment:
I'm unsure if this is a definitive list of parameters to check. In https://github.com/ethereum-optimism/client-pod/issues/210#issuecomment-1890339256, only `SubmissionInterval` and `FinalizationPeriodSeconds` are mentioned. I'm fairly sure `L2BlockTime` is important. But maybe not e.g. `StartingTimestamp`?

Reply:
[f6be092](https://github.com/ethereum-optimism/superchain-registry/pull/79/commits/f6be092a9e3a56672b737fd3e7f62281ad46d33c)